src/mame/subsino/subsino.cpp

```diff
@@ -1020,6 +1023,37 @@ void subsino_state::tisub_map(address_map &map)
 	map(0x15c00, 0x15dff).ram().w(FUNC(subsino_state::reel_ram_w<2>)).share("reel_ram.2");
 }
 
+void subsino_state::newhunterb_map(address_map &map)
+{
+	map(0x00000, 0x077ff).rom();
+	map(0x07800, 0x07fff).ram();
+	map(0x08000, 0x087ff).ram().w(FUNC(subsino_state::colorram_w)).share("colorram");
+	map(0x08800, 0x08fff).ram().w(FUNC(subsino_state::videoram_w)).share("videoram");
+
+	map(0x09000, 0x09002).r("ppi1", FUNC(i8255_device::read));
+	map(0x09004, 0x09006).r("ppi2", FUNC(i8255_device::read));
+
+	map(0x09008, 0x09008).w(FUNC(subsino_state::out_c_w));
+	map(0x09009, 0x09009).w(FUNC(subsino_state::out_b_w));
+	map(0x0900a, 0x0900a).w(FUNC(subsino_state::out_a_w));
+
+	map(0x0900c, 0x0900d).w("ymsnd", FUNC(ym3812_device::write));
+	map(0x0900e, 0x0900e).portr("INC");
+	map(0x0900f, 0x0900f).w(FUNC(subsino_state::tiles_offset_w));
+
+	map(0x09800, 0x09fff).ram();
+
+	map(0x10000, 0x13fff).rom();
+
+	map(0x150c0, 0x150ff).ram().share("reel_scroll.2");
+	map(0x15140, 0x1517f).ram().share("reel_scroll.1");
+	map(0x15180, 0x151bf).ram().share("reel_scroll.0");
+
+	map(0x15800, 0x159ff).ram().w(FUNC(subsino_state::reel_ram_w<0>)).share("reel_ram.0");
+	map(0x15a00, 0x15bff).ram().w(FUNC(subsino_state::reel_ram_w<1>)).share("reel_ram.1");
+	map(0x15c00, 0x15dff).ram().w(FUNC(subsino_state::reel_ram_w<2>)).share("reel_ram.2");
+}
```

Comment:
How different is this to the other address maps?  Can it be derived from a common map with just differences applied??

Reply:
@Osso13 Can you please help with this question?

Reply:
I'll have a look tomorrow (probably)